cmd/notary/keys.go

```diff
@@ -454,12 +455,21 @@ func removeKeyInteractively(keyStores []trustmanager.KeyStore, keyID string,
 	var storesByIndex []trustmanager.KeyStore
 
 	for _, store := range keyStores {
-		for keypath, keyInfo := range store.ListKeys() {
-			if filepath.Base(keypath) == keyID {
+		if strings.Contains(store.Name(), "Native keychain store") {
```

Comment:
I think we should be able to refactor this to not need to be split out -- in fact we probably don't need to `ListKeys()` at all here (unless I'm missing something)